app/Repositories/Dislike/DislikeRepository.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Hedonist\Repositories\Dislike;
+
+use Hedonist\Entities\Dislike\Dislike;
+use Prettus\Repository\Eloquent\BaseRepository;
+use Prettus\Repository\Criteria\RequestCriteria;
+use Prettus\Repository\Contracts\CriteriaInterface;
+use Illuminate\Database\Eloquent\Collection;
+
+class DislikeRepository extends BaseRepository implements DislikeRepositoryInterface
+{
+    public function model()
+    {
+        return Dislike::class;
+    }
+
+    public function save(Dislike $dislike): Dislike
+    {
+        $dislike->save();
+
+        return $dislike;
+    }
+
+    public function getById(int $id): ?Dislike
+    {
+        return Dislike::find($id);
+    }
+
+    public function findAll(): Collection
+    {
+        return Dislike::all();
+    }
+
+    public function findByCriteria(CriteriaInterface $criteria): Collection
+    {
+        return $this->getByCriteria($criteria);
+    }
+
+    public function findByUserAndPlace(int $userId, int $placeId): ?Dislike
+    {
+        return Dislike::where([
+            ['dislikeable_id', $placeId],
+            ['dislikeable_type', 'places'],
```

Comment:
Better to use `Place::class` instead of places. Use as refer [https://laravel.com/docs/5.6/eloquent-relationships#polymorphic-relations](https://laravel.com/docs/5.6/eloquent-relationships#polymorphic-relations).